panels/dev-info/ha-panel-dev-info.html

```diff
@@ -309,7 +310,16 @@ <h2>Log Details ([[selectedItem.level]])</h2>
   }
 
   formatTime(date) {
-    return window.hassUtil.formatTime(new Date(date * 1000));
+    const today = new Date().setHours(0, 0, 0, 0);
+    const dateTime = new Date(date * 1000);
+    const dateTimeDay = new Date(date * 1000).setHours(0, 0, 0, 0);
```

Comment:
we would use `dateTime.setHours(0, 0, 0, 0);` here but I think my code was fine already as we shouldn't define var's if we only need it once. `today` is more for a better readability

Reply:
ahh correct